resources/assets/js/app/services/SubscriptionService.js

```diff
@@ -0,0 +1,19 @@
+import {securedRequest} from 'app/services/RequestService';
+
+export const getSubscriptions = () => {
+    return securedRequest.get('/api/v1/subscriptions');
+};
+
+export const changeSubscriptionStatus = (id, status) => {
+    return securedRequest.put(`/api/v1/subscriptions/${id}/status`, {
+        status: status ? 'activate' : 'deactivate'
```

Comment:
send is_active with bool value instead of abstract status field with abstract activate/deactivate values